verilog/tools/ls/ls-wrapper.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021-2022 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VERILOG_TOOLS_LS_LS_WRAPPER_H
+#define VERILOG_TOOLS_LS_LS_WRAPPER_H
+
+#include <functional>
+
+#include "absl/status/status.h"
+#include "absl/strings/string_view.h"
+#include "common/lsp/json-rpc-dispatcher.h"
+#include "common/lsp/lsp-protocol.h"
+#include "common/lsp/message-stream-splitter.h"
+#include "verilog/tools/ls/verible-lsp-adapter.h"
+
+namespace verilog {
+
+// Output formatter for messages sent to LS client
+void JsonRpcDispatcherOutputFormatter(absl::string_view reply);
+
+// Class implementing the Language Server for Verilog
+class VerilogLanguageServer {
+ public:
+  static std::string GetVersionNumber() { return "0.0 alpha"; }
+
+  // Constructor preparing the callbacks for Language Server requests
+  VerilogLanguageServer();
+
+  // Runs the Language Server
+  absl::Status Run();
+
+  // Prints statistics of the current Language Server session
+  void PrintStatistics();
```

Comment:
Is `PrintStatistics()` modifying anything ? Can it be a `const` method ?

Reply:
Changed to const (same with InitializeRequestHandler)